assets/js/pages-page.js

```diff
@@ -367,6 +374,36 @@
         return false
     }
 
+    /*
+     * Triggered when the Duplicate button is clicked on the page form
+     */
+    PagesPage.prototype.onDuplicateObject = function(e) {
+        var self = this,
+            $button = $(e.target),
+            $form = $button.closest('form'),
+            parent = $button.data('parent') !== undefined ? $button.data('parent') : null,
```

Comment:
@der-On unless I'm mistaken, this will always be `null`. There doesn't seem to be a parent defined in the data for the duplicate button. I don't think this is needed anyway, as the parent will be copied from the original page, so you could probably remove this and any references to parent in here and the PHP handler.

Reply:
Yes, I've discovered right after making the PR that duplicating a nested page places it at the root level. Will adjust the PR to include the parent so nested pages get duplicated into the parent.